check/setup.go

```diff
@@ -235,7 +245,8 @@ func reassignPartition(zk ZkConnection, partitionID int32, replicas []int32, top
 	reassign := fmt.Sprintf(`{"version":1,"partitions":[{"topic":"%s","partition":%d,"replicas":[%s]}]}`,
 		topicName, partitionID, strings.Join(replicasStr, ","))
 
-	repeat = true
+	// Start new partition reassignment process
+	repeat := true
 	for repeat {
```

Comment:
I realize this is not part of your change, but I would think you'd want some kind of sleep in this loop.